pkg/chains/signing.go

```diff
@@ -37,13 +37,19 @@ type Signer interface {
 	Sign(ctx context.Context, obj objects.TektonObject) error
 }
 
+type MetricsRecorder interface {
+	RecordCountMetrics(MetricType string, ctx context.Context)
```

Comment:
Context should be the first parameter as per go's convention.

Reply:
Done